### PR TITLE
Fix typo in README ENV vars names

### PR DESCRIPTION
**Pull request by Mat001**
## Summary
- Typo in ENV variable name in README file

The "why", or other context.

## Issues
- https://optimizely.atlassian.net/browse/OASIS-7856

### CODE REVIEW

**Comment by keppel2** [comment]
Alphabetize. Thanks!

**Comment by keppel2** [comment]
Or you can do a separate PR. Alphabetize by property name. Thanks!

File: README.md [hunk 1]
```diff
@@ -85,7 +85,7 @@ Below is a comprehensive list of available configuration properties.
 |api.auth.ttl|OPTIMIZELY_API_AUTH_TTL|Time-to-live of issued access tokens. See: [Authorization Guide](https://docs.developers.optimizely.com/full-stack/docs/authorization)|
 |api.enableNotifications|OPTIMIZELY_API_ENABLENOTIFICATIONS|Enable streaming notification endpoint. Default: false|
 |api.enableOverrides|OPTIMIZELY_API_ENABLEOVERRIDES|Enable bucketing overrides endpoint. Default: false|
-|api.maxConns|OPTIMIZLEY_API_MAXCONNS|Maximum number of concurrent requests|
+|api.maxConns|OPTIMIZELEY_API_MAXCONNS|Maximum number of concurrent requests|
```
Comment (keppel2) [inline README.md:88]: `OPTIMIZELY`.

**Comment by Mat001** [comment]
Duplicate. Closing.

**Pull request closed without merging**